### PR TITLE
Feature: Accessibility

### PR DESCRIPTION
**Pull request by luftedar**
In this update:

- Update Accessibility:
  - **Page Title;** 
    Adequate and descriptive title added.
  - **Image Alt Attribute;** 
    Alternatives changes as none.
  - **Text Headings;** 
    Tags hierarchy updated
  - **Contrast;** 
    Changed contrast ratio of buttons. Checked with https://color.a11y.com/Contrast/.
  - **Resize;** 
    No issues found
  - **Interaction;** 
    No issues found
  - **Moving content;** 
    No issues found
  - **Multimedia;** 
    No issues found
  - **Basic structure;** 
    No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,13 +5,13 @@
   <meta charset="UTF-8" />
```
Comment (SajjadAhmad14) [inline index.html:5]: Please add a link to the corresponding line of code in the Pull Request which required changes.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aku Khan and Orcun Ugur, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (luftedar) [reply index.html:5]: Hello :wave: 
Thanks for your review. But I couldn't understand what you mean by this comment. Would you mind if you could clarify this one?

**Comment by luftedar** [comment]
**-Page Title Updated;**
The title of the page was not adequate and descriptive so I changed it.
[TitleLink](https://github.com/luftedar/My-Portfolio/blob/d03615fceedf0a58ee9add03b45453fb2f1cf5e2/index.html#L13)
**-Image Alt Attributes Changed;**
The Image tag's alternatives were not giving any information about the website. So alt attributes have changed.
[AltLink](https://github.com/luftedar/My-Portfolio/blob/4ef293910f37ca6e31f20dbaee7f3321957ddb7f/index.html#L34)
**-Text Headings;**
The header tags had hierarchy problems. So, the hierarchy changed.
[HeaderLink](https://github.com/luftedar/My-Portfolio/blob/d03615fceedf0a58ee9add03b45453fb2f1cf5e2/index.html#L30-L238)
**-Color Contrast;**
The color contrast for buttons had problems so I used 'filter: contrast(1.5)' for the problem. It solved according to https://color.a11y.com/Contrast/
[ButtonLink](https://github.com/luftedar/My-Portfolio/blob/d03615fceedf0a58ee9add03b45453fb2f1cf5e2/style.css#L258)
**-Resize;**
No issues found
**-Interaction;**
No issues found
**-Moving content;**
No issues found
**-Multimedia;**
No issues found
**-Basic structure;**
No issues found

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <link rel="preconnect" href="https://fonts.googleapis.com" />
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet" />
-  <title>Software Developer | Orçun Uğur's Portfolio</title>
+  <title>Orçun Uğur's Portfolio</title>
```
Comment (aakbarkhan) [inline index.html]: Change the title to be more informative.

File: style.css [hunk 3]
```diff
@@ -258,6 +258,7 @@ h1 {
   line-height: 24px;
   margin-top: 1.5rem;
   margin-bottom: 1.3rem;
+  filter: contrast(1.5);
```
Comment (aakbarkhan) [inline style.css:258]: Add the button contrast for more readibiility.

**Review by AshakaE — APPROVED** [review]
### Hi @luftedar  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**